.github/workflows/ci.yml

```diff
@@ -13,9 +13,11 @@ jobs:
             - uses: actions/setup-python@v4
               with:
                   python-version: '3.11'
-            - run: pip install -U ruff==0.0.284
+            - run: pip install -U ruff==0.0.284 mypy==1.5.1
             - name: Run ruff
               run: ruff docker tests
+            - name: Run mypy
+              run: mypy docker
```

Comment:
I'd suggest putting the files in the config and then only running `mypy`.
I'd also suggest copying, for example, this config, to get more in depth checking: https://github.com/aio-libs/aiohttp-jinja2/blob/master/.mypy.ini
Commenting out any options that you don't want to deal with yet, so they can be added in later (note you can also do config options per module).

Reply:
Or maybe we should go with mypy strict in one go?

Reply:
strict just enables about half the options listed in that config (there are several other useful ones not enabled by strict), so I'd still look at the whole thing. And if you don't want to resolve everything in 1 PR, then more finegrained settings is going to help.

Reply:
I see, I'll take a look. I'll see if CI passes once I convert this PR as ready to review. But I don't think maintainers will see..